scene/resources/animation.h

```diff
@@ -455,6 +455,7 @@ class Animation : public Resource {
 	void track_set_interpolation_type(int p_track, InterpolationType p_interp);
 	InterpolationType track_get_interpolation_type(int p_track) const;
 
+	void bezier_track_insert_key_with_handle_mode(int p_track, double p_time, real_t p_value, const Vector2 &p_in_handle, const Vector2 &p_out_handle, HandleMode p_handle_mode);
```

Comment:
`HandleMode` is `TOOLS_ENABLED` (editor) only, so this doesn't compile in export templates.

If this is only needed for the editor, it should also be enclosed in `#ifdef TOOLS_ENABLED`, but then it can't be exposed in `_bind_methods` as the scripting API shouldn't differ between tools and non-tools.

Reply:
oh I see. Well, it should not be exposed in the editor, but I figured it would have to be bound, such that it can be called from a signal, right?

Reply:
To use as a signal callback it doesn't need to be bound, you can use `callable_mp`.
But in this PR you're using it for UndoRedo and that one currently still requires method bindings, it doesn't support `callable_mp`.

So ideally if there's a way to achieve what you're doing while keeping the method within the AnimationBezierEditor, that might be best. Otherwise it might need some creative API rethinking (possibly exposing HandleMode to non-tools if there's no good alternative).

Reply:
Ok, makes sense. The easiest is to move it back to the bezier editor then. Thank you, akien! :)